pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/ClusterIntegrationTestUtils.java

```diff
@@ -903,25 +900,21 @@ private static void comparePinotResultsWithExpectedValues(Set<String> expectedVa
   }
 
   private static String removeTrailingZeroForNumber(String value, String type) {
+    String upperCaseType = StringUtils.toUpperCase(type);
```

Comment:
```suggestion
    String upperCaseType = type.toUpperCase();
```

Reply:
`type.toUpperCase()` is not null safe, though we aren't sending the `null` value.
For future perspective changed it to `String upperCaseType = StringUtils.upperCase(type);`